### PR TITLE
better function name

### PR DESCRIPTION
**Pull request by charlierudolph**
@allewun @kevgo 

Make exit_with_error only exit and introduce a new function for printing a new line and exiting with error.

### CODE REVIEW

File: src/helpers/terminal_helpers.sh [hunk 1]
```diff
@@ -96,15 +96,14 @@ function echo_usage_header {
...
 function exit_with_error {
-  echo
   exit 1
 }
 
 
-# Exits the currently running script with a success response code
-function exit_with_success {
+# Prints a newline and exits with an error
+function echo_and_exit_with_error {
```
Comment (allewun) [inline src/helpers/terminal_helpers.sh]: How about `newline_and_exit_with_error`? echo is an implementation detail, I suppose there are other ways to print a newline

Comment (kevgo) [inline src/helpers/terminal_helpers.sh]: +1

**Comment by charlierudolph** [comment]
@allewun @kevgo all comments addressed

**Comment by allewun** [comment]
:ship:

**Comment by kevgo** [comment]
:ship:

**Pull request closed without merging**